libs/sdk-ui/src/base/react/usePromiseCache.ts

```diff
@@ -88,7 +91,12 @@ export function usePromiseCache<TParams, TResult, TError>(
 
         return () => {
             preventResetPromisesRef.current = !!preventResetPromises;
-            usedPromiseCache.cancel(newParams[0]);
+            // only canceling the promise when the search string is changed
+            const { title: currentSearchText = "" } = newParams[0] as { title: string };
```

Comment:
I would expect that this handling of params will be much more generic as this promiseCache is quite generic piece of code

Reply:
Hi @ivan-nejezchleb, I have updated this. Can you please review the changes again? Thank you!